Flagsmith.FlagsmithClient/Feature.cs

```diff
@@ -0,0 +1,30 @@
+using Newtonsoft.Json;
+
+namespace Flagsmith
+{
+    [JsonObject(MemberSerialization.OptIn)]
+    public class Feature
```

Comment:
I'm not too sure why we need this class in the Client project? The python client just uses the feature id and feature name rather than needing a whole class to hold the information.

Reply:
as per API JSON response , we have an object of feature , so i find it convenient to create its separate class.